homeassistant/components/mealie/strings.json

```diff
@@ -29,6 +29,7 @@
     "error": {
       "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "ingress_url": "Ingress URL's are not supported. Use the direct host and port.",
```

Comment:
Should we explain this more? Would, for example, your dad know what ingress is? Or a direct host or port?

Reply:
I only learnt what ingress url's are this week, added a more explanative error with directions on where to find the port.